docs/content/3.components/calendar.md

```diff
@@ -21,21 +21,34 @@ Use the `v-model` directive to control the selected date.
 
 ::component-code
 ---
+cast:
+  modelValue: DateValue
+ignore:
+  - modelValue
+external:
+  - modelValue
+props:
+  modelValue: [2022, 2, 3]
 ---
 ::
 
-<!-- TODO: Add example with default value -->
```

Comment:
Can you add the example with `default-value` here?

Reply:
Similar to https://ui3.nuxt.dev/components/input-number#usage

Reply:
Done